src/Error/Renderer/ConsoleExceptionRenderer.php

```diff
@@ -68,35 +68,60 @@ public function __construct(Throwable $error, ?ServerRequestInterface $request,
      */
     public function render()
     {
+        $exceptions = [$this->error];
+        $previous = $this->error->getPrevious();
+        while ($previous !== null) {
+            $exceptions[] = $previous;
+            $previous = $previous->getPrevious();
+        }
         $out = [];
-        $out[] = sprintf(
-            '<error>[%s] %s</error> in %s on line %s',
-            get_class($this->error),
-            $this->error->getMessage(),
-            $this->error->getFile(),
-            $this->error->getLine()
-        );
+        foreach ($exceptions as $i => $error) {
```

Comment:
Don't want want to render in reverse order?

Reply:
I would want to read the initial exception which started the chain.

Reply:
I guess it depends how the display looks. I could see a callstack style making sense so you can see where it came from.

Reply:
Let's see how it feels.